runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/SimpleExecutionState.java

```diff
@@ -149,11 +149,40 @@ public String getLullMessage(Thread trackedThread, Duration millis) {
     return message.toString();
   }
 
+  @VisibleForTesting
+  public String getBundleLullMessage(Thread trackedThread, Duration millis) {
+    // TODO(ajamato): Share getLullMessage code with DataflowExecutionState.
+    String userStepName =
+        this.labelsMetadata.getOrDefault(MonitoringInfoConstants.Labels.PTRANSFORM, null);
+    StringBuilder message = new StringBuilder();
+    message.append("Operation ongoing");
+    if (userStepName != null) {
+      message.append(" in bundle ").append(userStepName);
```

Comment:
the bundle will process across many steps, so this seems confusing.
It seems like instead it should show the fused stage here

Reply:
Done

Reply:
Removed the step info here.